modules/agent-configuration-parameters.adoc

```diff
@@ -0,0 +1,120 @@
+
+|metadata:
+|Kubernetes resource `ObjectMeta`, from which only the `name` parameter is consumed.
+|Object
+
+|metadata:
+  name:
+|The name of the cluster.
+DNS records for the cluster are all subdomains of `{{.metadata.name}}.{{.baseDomain}}`.
+|String of lowercase letters and hyphens (`-`), such as `dev`.
+|====
+
+
+[id="agent-configuration-parameters-optional_{context}"]
+== Optional configuration parameters
+
+Optional Agent configuration parameters are described in the following table:
+
+.Optional parameters
+[cols=".^2l,.^3,.^5a",options="header"]
+|====
+|Parameter|Description|Values
+
+|rendezvousIP:
+|The IP address of the node that performs the bootstrapping process as well as running the `assisted-service` component.
+You must provide the rendezvous IP address when you do not specify at least one host's IP address in the `networkConfig` parameter.
+If this address is not provided, one IP address is selected from the provided hosts' `networkConfig`.
+|IPv4 or IPv6 address.
+IPv6 is supported only on bare metal platforms.
+
+|bootArtifactsBaseURL:
+|(needs info)
+|(needs accepted/valid values)
+
+|additionalNTPSources:
+|A list of Network Time Protocol (NTP) sources to be added to all cluster hosts, which are added to any NTP sources that are configured through other means.
+|List of hostnames or IP addresses.
+
+|hosts:
+|Host configuration.
+An optional list of hosts.
+The number of hosts defined must not exceed the total number of hosts defined in the `install-config.yaml` file, which is the sum of the values of the `compute.replicas` and `controlPlane.replicas` parameters.
+|An array of host configuration objects.
+
+|hosts:
+  hostname:
+|Hostname.
+Overrides the hostname obtained from either the Dynamic Host Configuration Protocol (DHCP) or a reverse DNS lookup.
+Each host must have a unique hostname supplied by one of these methods, although configuring a hostname through this parameter is optional.
+|(needs accepted/valid values)
+
+|hosts:
+  interfaces:
+|Provides a table of the name and MAC address mappings for the interfaces on the host.
+If a `NetworkConfig` section is provided in the `agent-config.yaml`, this table must be included and the values must match the mappings provided in the `NetworkConfig` section.
+|An array of host configuration objects.
+
+|hosts:
+  interfaces:
+    name:
+|The name of an interface on the host.
+|(needs accepted/valid values)
+
+|hosts:
+  interfaces:
+    macAddress:
+|The MAC address of an interface on the host.
+|A MAC address such as the following example: `00-B0-D0-63-C2-26`
+
+|hosts:
+  rootDeviceHints:
+|Enables provisioning of the {op-system-first} image to a particular device.
+It examines the devices in the order it discovers them, and compares the discovered values with the hint values.
+It uses the first discovered device that matches the hint value.
+This is the device that the operating system will be written on during installation.
+|(needs accepted/valid values)
```

Comment:
I don't think need to list accepted/valid values here, it should just refer to other section in the docs that describes rootDeviceHints e.g. https://docs.openshift.com/container-platform/4.13/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#root-device-hints_ipi-install-installation-workflow

Reply:
I still think there might be a need to define the type of data that should be used with this parameter (and all others, even if we don't want to specify which specific values are accepted). Would this be an accurate entry to include here?

`A dictionary of key-value pairs. For more information, see Root device hints.`

With a link to the section you provide in your comment

Reply:
Sure